optimizely/project_config.py

```diff
@@ -106,12 +106,19 @@ def __init__(self, datafile, logger, error_handler):
         )
 
     self.feature_key_map = self._generate_key_map(self.feature_flags, 'key', entities.FeatureFlag)
+
+    # Hash containing experiment Ids that exist in any feature
```

Comment:
nit. Dict and not hash. 

Also, right message will be `Dict containing map of experiment ID to feature ID ... `